app/helpers/catalog_helper.rb

```diff
@@ -96,8 +96,9 @@ def display_access_control_level(document, metadata_colls)
     access_group = document['read_access_group_ssim'] # "public" > "local" > "dams-curator" == "dams-rci" == default
     view_access = 'Curator Only'
     if access_group
-
-      if metadata_colls.include?(document['id']) || metadata_display?(rights_data(document))
+      if metadata_colls.include?("#{document['id']}true")

```

Comment:
What is the `true` part here? Is that part of the query response, or?

Reply:
It's just a tag there to indicate that's a mixed object collection build from this line
https://github.com/ucsdlib/damspas/pull/494/files#diff-956e46b33fb5307990c4e1a4b5fd86ccR329

Reply:
Ah, I see. Would it be worth adding a comment or something for that? Now that you point it out that makes sense, but it wasn't obvious to me initially that was part of the solution. Especially since `true` is a keyword, so my brain defaults to that i guess.